scipy/linalg/_sketches.py

```diff
@@ -1,24 +1,22 @@
 """ Sketching-based Matrix Computations """
-
-# Author: Jordi Montes <jomsdev@gmail.com>
-# August 28, 2017
-
```

Comment:
Maybe keep this info.

Reply:
Fixed in most recent commit. I was unclear on the standard so I appreciate this comment.